Include fan-out operators in the operator index (#27028)
akka/akka: akka-docs/src/main/paradox/stream/operators/index.md

|Source/Flow|<a name="zipwith"></a>@ref[zipWith](Source-or-Flow/zipWith.md)|Combines elements from multiple sources through a `combine` function and passes the returned value downstream.|
|Source/Flow|<a name="zipwithindex"></a>@ref[zipWithIndex](Source-or-Flow/zipWithIndex.md)|Zips elements of current flow with its indices.|

## Fan-out operators

These have one input and multiple outputs. They might route the elements between different outputs, or emit elements on
multiple outputs at the same time.

> There is currently no 'fluent' API available for the fan-out operators, to use them you will have to use the @ref[Graph DSL](../stream-graphs.md#constructing-graphs).

Reviewer: ```suggestion
A 'fluent' API for the fan-out operators is not yet available, until then use the @ref[Graph DSL](../stream-graphs.md#constructing-graphs).
```
